torchci/rockset/commons/__sql/commit_jobs_query.sql

```diff
@@ -67,15 +62,23 @@ WITH job as (
             PARSE_TIMESTAMP_ISO8601(job.job.stopped_at)
         ) as duration_s,
         -- Classifications not yet supported
-        null,
-        null,
-        null,
-        null,
     FROM
         circleci.job job
     WHERE
         job.pipeline.vcs.revision = :sha
         AND CONCAT(job.organization.name, '/', job.project.name) = :repo
+),
+-- Use a subquery for classifications to ensure we can do a lookup join, which greatly improves query performance.
```

Comment:
I'm curious about how this improves performance. Very cool though!

Reply:
the LEFT JOIN on classification performs a full table scan, which is slow. By separating out the classification and forcing a lookup join, we improve the query performance by using an index instead of doing the table scan.

I used the query performance profiler and the guidance in https://rockset.com/docs/query-composition/ to decide what to do